### ISSUE TITLE
Conda-forge install seems broken

### ISSUE THREAD

**Issue by tomkom** [mdbartos/pysheds#223]
I have been trying to install pysheds on MAC OSX (M1) using the standard:

`conda install -c conda-forge pysheds`

Resulting in: >> PackagesNotFoundError: The following packages are not available from current channels:

This install currently works perfectly on Linux, but fails on Mac.

**Comment by mdbartos** [comment]
Interesting. What version of python are you using?

Maybe the conda forge repo might hold some answers?:
https://github.com/conda-forge/pysheds-feedstock

The mac build appears on the anaconda site:
https://anaconda.org/conda-forge/pysheds

I develop on an M1 mac, so I know it works on that platform, but I use pip for handling installs.

**Comment by tomkom** [comment]
Yes, I checked all those. Interestingly, no problem to install on Linux... Will try to diagnose more.

**Comment by giorgionicoletti** [comment]
Hey, I'm encountering the same issue (with an M2 Mac). Any idea why this is happening?

**Comment by pkmn99** [comment]
This is probably because pysheds on conda does not have an arm build named "osx-arm64", while osx-64 is for intel cpu

**Comment by cheginit** [comment]
The issue with `osx-arm64` can be solved following the instruction [here](https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/#how-to-add-a-osx-arm64-build-to-a-feedstock). 
Essentially, [this](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/migrations/osx_arm64.txt) file needs to be edited by adding `pysheds` to the list, which then triggers a bot to make all the necessary changes to the `pysheds` `conda-forge` receipt repo and open a PR. Then, the maintainer of the `conda-forge` receipt repository for `pysheds` just needs to merge that PR.

@mdbartos If interested, I can add `pysheds` to the `osx_arm64` file, so you can merge the PR.
